File: PRAK402-2210817210015-MuhammadIrfan.c
#include <stdio.h>
int main(){
    int bil, a;
    printf("INPUT  : ");
    scanf("%d", &bil);
    printf("OUTPUT : \n");
    for(a=1; a<=bil; a++){
        if(a%2==1){
            printf("%d ", a);}
        else{}
    }
    printf("\n");
    for(a=bil; a>=1; a--){
        if(a%2==0){
            printf("%d ", a);}
        else{}
    }
}

File: Python/PRAK402-2210817210015-MuhammadIrfan.py
bil = int(input('INPUT  : '))
print('OUTPUT :')
for i in range (1, bil):
    if i%2!=0:
        print(i, end=' ')
print('\n')
while bil > 0:
    if bil%2==0:
        print(bil, end=' ')
    bil-=1